### ISSUE TITLE
Support deep sleep wakeup on ESP32

### ISSUE THREAD

**Issue by JeffinChang** [wokwi/wokwi-features#441]
**Is your feature request related to a problem? Please describe.**
Attempt to run ESP32 example: TimerWakeUP.ino but it seems Wokwi did not support deep sleep wakeup, as there is NO wakeup after going to sleep. 

**Describe the solution you'd like**
Need support deep sleep wakeup on ESP32

**Comment by urish** [comment]
Thanks for creating this feature request! I opened it for [voting](https://wokwi.com/features?sort=newest#feature-1429239393)

**Comment by Phetdanai** [comment]
Good, @urish.

**Comment by tyeth** [comment]
I'm interested in testing deep sleeping wifi based sensors, running arduino style sketches, something like the Adafruit WipperSnapper firmware. I'd be doing both browser and CI stuff, but eventually mostly just CI.
I'm also very keen to play with deep sleep on circuitpython which is more for rapid prototyping in the browser, particularly with connected displays.

**Comment by huseenhejazy7** [comment]
good

**Comment by urish** [comment]
Reproduction from @petermm:

https://wokwi.com/projects/457488000233051137

**Comment by petermm** [comment]
Yeah, sorry for being slow - was looking at the gpio interrupt things - for a complete example, but basic support is a great first step!

**Comment by urish** [comment]
Deep sleep wakeup is now supported!
